src/Workspaces/Core/Portable/Rename/ConflictEngine/DeclarationConflictHelpers.cs

```diff
@@ -18,7 +16,7 @@ public static IEnumerable<Location> GetMembersWithConflictingSignatures(IMethodS
                                             .OfType<IMethodSymbol>()
                                             .Where(m => !m.Equals(renamedMethod) && m.Arity == renamedMethod.Arity);
 
-            var signatureToConflictingMember = new Dictionary<List<ITypeSymbol>, IMethodSymbol>(new ConflictingSignatureComparer());
+            var signatureToConflictingMember = new Dictionary<ImmutableArray<ITypeSymbol>, IMethodSymbol>(ConflictingSignatureComparer.Instance);
```

Comment:
We're using a custom comparer, so the type's hashcode doesn't actually matter.  This particular change just makes it easier to reason about the code.